Mathlib/GroupTheory/GroupAction/Blocks.lean

```diff
@@ -381,6 +381,93 @@ theorem IsBlockSystem.of_normal {N : Subgroup G} [N.Normal] :
 
 end Normal
 
+section Stabilizer
+
+/- For transitive actions, construction of the lattice equivalence
+  `block_stabilizerOrderIso` between
+  - blocks of `MulAction G X` containing a point `a ∈ X`,
+  and
+  - subgroups of G containing `stabilizer G a`.
+  (Wielandt, th. 7.5) -/
+
+/-- The orbit of a under a subgroup containing the stabilizer of a
+ is a block -/
+theorem IsBlock.of_orbit' {H : Subgroup G} {a : X} (hH : stabilizer G a ≤ H) :
```

Comment:
Why is this primed? Did you want to call it `isBlock_orbit'`?

Reply:
Because there used to be `IsBlock.of_orbit` which says `IsBlock G (MulAction.orbit G a)` (so with H = top), and I don't remember why that theorem is not in this PR. (That may be a bug of me splitting one file into 5 PR).

Reply:
Isn't that theorem called `isBlock_orbit`?

Reply:
You're right. Here is why I'm confused. I am tempted to use dot notation, and since that applies to `IsBlock` property, to keep the `IsBlock` namespace. But sometimes my good intentions don't work as well.
So `isBlock_orbit'` should at least be consistent with the first one.

Reply:
I changed the name to `IsBlock.of_orbit`.